nisystemlink/clients/core/_internal/_path_constants.py

```diff
@@ -21,9 +21,9 @@ class PathConstants(metaclass=ClasspropertySupport):
 
     PRODUCT_NAME = "Skyline"
 
-    _application_data_directory = None  # type: Optional[pathlib.Path]
+    _application_data_directory: Optional[pathlib.Path] = None
```

Comment:
Note: The code changes are to resolve flake8 thinking some typing imports are unused.